docs/source/more_types.rst

```diff
@@ -824,6 +824,20 @@ extra items is compatible with a narrower TypedDict, assuming item
 types are compatible (*totality* also affects
 subtyping, as discussed below).
 
+A TypedDict object is not compatible with the regular ``Dict[...]``
+type, since ``Dict`` allows arbitrary keys to be added and removed,
+unlike TypedDict. However, any TypedDict object is compatible with
```

Comment:
I don't like the term "compatible". I know we use it all over the place, but I always have to look for clues elsewhere to tell me whether it means "subtype" or "supertype". Maybe add "(i.e., is a subtype of)" at the end of this line?